src/translations/en.json

```diff
@@ -15,8 +15,6 @@
   },
   "state": {
     "default": {
-      "on": "On",
-      "off": "Off",
```

Comment:
Used nowhere else? 😱

Reply:
Yes, it's only use for unknown and unavailable